WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -40,26 +41,26 @@
         </div>
       </div>
     <% else %>
-      <% if @competition.isConfirmed? && @competition.showAtAll? %>
+      <% if is_actually_confirmed && @competition.showAtAll? %>
```

Comment:
Does the commit message here: https://github.com/jfly/worldcubeassociation.org/commit/71454c814d6af91d6539fe3139f61288fc9fe8a6 make sense?

I was a bad boy and didn't write a test for this, under the argument that it's "just" ui code (something I haven't been very rigorous about testing). This is subtle enough that maybe it deserves a test. What do you think?